src/main/java/mate/academy/spring/validator/EmailValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^((([0-9A-Za-z]{1}[-0-9A-z\\.]"
+            + "{0,30}[0-9A-Za-z]?)|([0-9А-Яа-я]{1}[-0-9А-я\\.]{0,30}[0-9А-Яа-я]?))"
+            + "@([-A-Za-z]{1,}\\.){1,}[-A-Za-z]{2,})$";
+
+    @Override
+    public boolean isValid(String email,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        if (email.isEmpty()) {
```

Comment:
add null check please
annotations are not validated in a strong order
so you may still have NPE